spec/features/apps/new_app_spec.rb

```diff
@@ -0,0 +1,63 @@
+require 'rails_helper'
+
+RSpec.describe "New Application", type: :feature do
+
+  before :each do
+    @shelter1 = Shelter.create(name: "the lab", address: "666 dog Ave", city: "ruff town", state: "DG", zip: "12345")
```

Comment:
a lot of code is exicuted before each. basically just selecting 2 pets and adding to favorites, then going to favorites. When we have time you should give me a rundown on factory bot